main.go

```diff
@@ -48,6 +49,7 @@ type Alert struct {
 	GeneratorURL string
 	StartsAt     string
 	ValueString  string
+	ExternalURL  string
```

Comment:
I think this maybe should this be GeneratorURL? I don't see a reference to ExternalURL being used in the [alert object](https://github.com/prometheus/alertmanager/blob/v0.25.0/template/template.go#L251) that alertmanager will send.

Reply:
Honestly, I did not dig into them that deep because I wanted only to include functions that were an option in [Grafana Template functions] (https://grafana.com/docs/grafana/latest/alerting/fundamentals/annotation-label/template-functions/). These being excluded were additional functions that Prometheus had listed. Grafana did not list them as an option. I figured for consistency purposes, excluding them would be best to keep the template functions that the bridge supports the same as Grafana. Plus, it simplifies our documentation in the ReadMe. Thoughts?

Reply:
I just replied in the other thread, but I think we could just drop this and state that we're using the [Prometheus template functions ](https://prometheus.io/docs/prometheus/latest/configuration/template_reference/) since that is what is really happening under the covers. My guess is that the Grafana implementation is doing the same thing